pkg/controller/controller.go

```diff
@@ -130,7 +130,8 @@ func (c *Controller) ProcessNextPodItem() bool {
 		return false
 	}
 	defer c.podQueue.Done(key)
-	pod, err := c.podFromKey(key)
+	//pod, err := c.podFromKey(key)
+	pod, err := podFromKey(c.podInformer, key)
```

Comment:
do we plan on any unit tests calling `podFromKey` or `pvcFromKey`? If so then they need to be exported. But my guess is no in which case they can remain methods.